functional-tests/src/test/groovy/specs/traits/Users.groovy

```diff
@@ -0,0 +1,37 @@
+/*
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package traits
+
+/**
+ * Methods to manage user credentials.
+ */
+trait Users {
+  Map env = System.getenv()
+  Map getUserOne() {
+    [username:env['GWELLS_USERNAME'], password:env['GWELLS_PASSWORD']]
```

Comment:
This is a catch-all user with ALL roles and groups.  
This user should eventually not need to exist, but I ran into some issues where the "Registry" user didnt have all of the permissions i expected.  Maybe just need to double check how I set up the user in keycloak, or maybe there is an actual role/code issue. 

Can talk in person, when the time comes to try the tests in openshift.